src/components/Comments/ContainerComment.js

```diff
@@ -201,20 +201,20 @@ export const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row',
     paddingBottom: 14,
-    borderLeftColor: isLast ? 'transparent' : colors.gray1,
+    borderLeftColor: isLast ? 'transparent' : COLORS.gray9,
     borderLeftWidth: 1
   }),
   seeRepliesText: {
-    color: colors.blue
+    color: COLORS.blue
   },
   connector: {
     width: 10,
     height: 5,
     borderLeftWidth: 1.5,
     borderBottomWidth: 1.5,
     // borderBottomLeftRadius: 1,
-    borderLeftColor: colors.gray1,
-    borderBottomColor: colors.gray1,
+    borderLeftColor: COLORS.gray9,
+    borderBottomColor: COLORS.gray9,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderLeftColor and borderBottomColor properties in the style objects have been updated to use `COLORS.gray9` instead of `colors.gray1`. The color property for seeRepliesText has also been updated to use `COLORS.blue` instead of `colors.blue`. These changes are consistent with the PR summary and improve maintainability by centralizing color definitions.
```diff
-    borderLeftColor: isLast ? 'transparent' : colors.gray1,
+    borderLeftColor: isLast ? 'transparent' : COLORS.gray9,
...
-    color: colors.blue
+    color: COLORS.blue
...
-    borderLeftColor: colors.gray1,
-    borderBottomColor: colors.gray1,
+    borderLeftColor: COLORS.gray9,
+    borderBottomColor: COLORS.gray9,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->